packages/conf-qt/conf-qt.5.2.1/opam

```diff
@@ -9,6 +9,13 @@ dev-repo: "git+https://github.com/Kakadu/lablqml.git"
 
 #setenv: [ PATH += "/usr/lib64/qt5/bin:/usr/lib/qt5/bin"] #{ os-distribution = "alpine" | os-distribution = "centos" | os-distribution = "fedora" }
 
+# MacOS uses weird installation locations, and we need to patch the environment. Issues:
+# 1) Opam warns that this feature should be used only by compilers
+# 2) There is no way to apply setenv conitionally (guards are not allowed here).
+setenv: [
+  [ PKG_CONFIG_PATH += "/usr/local/opt/qt@5/lib/pkgconfig" ] # { os="macos" } #not allowed here
```

Comment:
This is just for homebrew though, right?

Reply:
Maybe. I have never seriously used a Mac, so I apply (kind of random) changes are just to make CI happy